app/uk/gov/hmrc/exports/repositories/SubmissionRepository.scala

```diff
@@ -43,13 +43,17 @@ class SubmissionRepository @Inject()(implicit mc: ReactiveMongoComponent, ec: Ex
   override def indexes: Seq[Index] = Seq(
     Index(Seq("actions.conversationId" -> IndexType.Ascending), unique = true, name = Some("conversationIdIdx")),
     Index(Seq("eori" -> IndexType.Ascending), name = Some("eoriIdx")),
-    Index(Seq("eori" -> IndexType.Ascending, "action.requestTimestamp" -> IndexType.Descending), name = Some("actionOrderedEori"))
+    Index(
```

Comment:
We probably need another index for your sort field

Reply:
Done - can you check @ed0906

Reply:
Looks good, presume thats the only field we need to sort on for the moment right?

I think `status` needs one too actually

Reply:
Add an index for status or the ability to sort-by?   That can be added when we switch to using the new findDeclaration service I guess